gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=6.1.0-SNAPSHOT
-iexecCommonVersion=5.5.0-SNAPSHOT
+version=6.0.3-NEXT-SNAPSHOT
```

Comment:
Backward version to make it compatible with latest versioning system.

Reply:
Noted, thanks!